packages/mjml/src/index.js

```diff
@@ -46,4 +49,19 @@ registerComponent(CarouselImage)
 
 registerDependencies(require('./dependencies'))
 
+
+try {
+  const mjmlConfig = fs.readFileSync(path.join(process.cwd(), '.mjmlconfig'));
+  const custom_comps = JSON.parse(mjmlConfig).packages
+
+  custom_comps.forEach((compPath) => {
+    const requiredComp = require(path.join(process.cwd(), compPath))
+    registerComponent(requiredComp.default || requiredComp)
+  })
+} catch(e) {
+  if (e.code !== 'ENOENT') {
```

Comment:
We might need a better error handling for each possible cases ?

Reply:
Actually ENOENT is the case where .mjmlconfig file does not exist
In this case we don't want to consider this as an error
For any other error we log it, but i haven't identified specific error cases